pkg/acceptance/helpers/secret_client.go

```diff
@@ -71,7 +71,7 @@ func (c *SecretClient) CreateWithBasicAuthenticationFlow(t *testing.T, id sdk.Sc
 func (c *SecretClient) CreateWithGenericString(t *testing.T, id sdk.SchemaObjectIdentifier, secretString string) (*sdk.Secret, func()) {
 	t.Helper()
 	ctx := context.Background()
-	request := sdk.NewCreateWithGenericStringSecretRequest(id, secretString)
+	request := sdk.NewCreateWithGenericStringSecretRequest(id, secretString).WithOrReplace(true)
```

Comment:
Is this needed? I'm guessing it was needed because in some of the tests you're calling Create too many times?